### ISSUE TITLE
Blask: Overlapping images on homepage

### ISSUE THREAD

**Issue by RealCarFax** [Automattic/themes#1236]
## Steps to replicate

Two screenshots for examples:
https://cloudup.com/cZHhD_GOh_k
https://cloudup.com/cnSpFAX9RBy

## Result
Images (portfolio project featured images) are sometimes appearing stacked on the homepage. Happens regardless of browser and regardless of the Infinite Scroll option chosen.

## Expected
The images should appear in a masonry grid without overlap.

When I open the inspector console in Firefox that reloads the Javascript and fixes the overlapping images it seems.

* User report: #14486317-hc
* Follow-up ticket: #2263588-zd

**Comment by danieldudzic** [comment]
Hey @RealCarFax 

I was able to observe the issue happening only once, here: https://harryelks.com/ - after that I was unable to replicate.

It seems to be quite intermittent.

Perhaps the JS wizard, @jeffikus would be able take a look?

**Comment by kavyagokul** [comment]
Also reported in 37481336-hc

This only happens when lazy loading for images is on. Turning lazy loading fixes this.
